app/controllers/api/application_controller.rb

```diff
@@ -11,4 +11,9 @@ def verify_json_request
   def verify_json_or_csv_request
     not_found unless request.format.json? || request.format.csv?
```

Comment:
do you think it is better to default to one (json probably)

Reply:
I removed the method as it's not used.